### PR TITLE
get-work-json & submit-work-json

### PR DESCRIPTION
**Pull request by HAOYUatHZ**

### CODE REVIEW

File: api/miner.go [hunk 1]
```diff
@@ -16,25 +29,63 @@ func (a *API) getWork() Response {
...
+	}
+
+	bh := &types.BlockHeader{
+		Version:           req.BlockHeader.Version,
+		Height:            req.BlockHeader.Height,
+		PreviousBlockHash: req.BlockHeader.PreviousBlockHash,
+		Timestamp:         req.BlockHeader.Timestamp,
+		Nonce:             req.BlockHeader.Nonce,
+		Bits:              req.BlockHeader.Bits,
+		BlockCommitment:   blockCommitment,
```
Comment (Paladz) [inline api/miner.go]: BlockCommitment: *req.BlockHeader.BlockCommitment

File: api/miner.go [hunk 2]
```diff
@@ -52,7 +103,44 @@ func (a *API) GetWork() (*GetWorkResp, error) {
...
+
+	return &GetWorkJSONResp{
+		BlockHeader: &BlockHeaderJSON{
+			Version:           bh.Version,
+			Height:            bh.Height,
+			PreviousBlockHash: bh.PreviousBlockHash,
+			Timestamp:         bh.Timestamp,
+			Nonce:             bh.Nonce,
+			Bits:              bh.Bits,
+			BlockCommitment:   blockCommitment,
```
Comment (Paladz) [inline api/miner.go]: BlockCommitment:  &bh.BlockCommitment

**Comment by boomyl** [comment]
/:strong

**Comment by zxcvnnm** [comment]
an0909228155@gmail.com.  2534

**Pull request merged**